web/modules/custom/joinup_core/src/WorkflowHelperInterface.php

```diff
@@ -11,6 +12,51 @@
 interface WorkflowHelperInterface {
 
   /**
+   * Returns the available transition states labels of an entity for given user.
+   *
+   * If no user is passed, the logged in user is checked. If no user is logged
+   * in, an anonymous account is passed.
+   *
+   * @param \Drupal\Core\Entity\FieldableEntityInterface $entity
+   *   The entity with the states.
+   * @param \Drupal\Core\Session\AccountInterface|null $user
+   *   The account interface object. Can be left empty.
+   *
+   * @return array
+   *   An array of transition state labels.
+   */
+  public function getAvailableStatesLabels(FieldableEntityInterface $entity, AccountInterface $user = NULL);
```

Comment:
Now that we are touching this, we can rename these `$user` to the more accepted `$account` in all these instances.